[docs][candi] Added note about node networks in noProxy parameter

Suselz opened the pull request:
## Description
Added additional information to clarify that all node networks must be included in the global noProxy parameter. 
This ensures users correctly configure their settings to avoid connectivity issues.


## Why do we need it, and what problem does it solve?
This complements and eliminates confusion among users.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
```changes
section: docs
type: feature 
summary: Added note about node networks in noProxy parameter
impact_level: low
```

z9r5 commented on candi/openapi/cluster_configuration.yaml:
```diff
@@ -229,6 +229,8 @@ apiVersions:
 
               For wildcard domains, use a domain name with a dot prefix, e.g., ".example.com".
+
+              > **Caution!** You must fill in the parameter with a list of subnets that are used in the cluster.
```
```suggestion
              > **Caution.** You must fill in the parameter with a list of subnets that are used in the cluster.
```

z9r5 commented on candi/openapi/doc-ru-cluster_configuration.yaml:
```diff
@@ -108,3 +108,5 @@ apiVersions:
 
               Для настройки wildcard-доменов используйте написание вида ".example.com".
+              
+              > **Внимание!** Требуется заполнить параметр списком подсетей, которые используются на узлах.
```
```suggestion
              > **Внимание.** Требуется заполнить параметр списком подсетей, которые используются на узлах.
```

z9r5 commented on candi/openapi/doc-ru-cluster_configuration.yaml:
```diff
@@ -108,3 +108,5 @@ apiVersions:
 
               Для настройки wildcard-доменов используйте написание вида ".example.com".
+              
+              > **Внимание.** Требуется заполнить параметр списком подсетей, которые используются на узлах.
```
```suggestion
              > **Внимание.** Укажите список подсетей, которые используются на узлах.
```

z9r5 commented on candi/openapi/cluster_configuration.yaml:
```diff
@@ -229,6 +229,8 @@ apiVersions:
 
               For wildcard domains, use a domain name with a dot prefix, e.g., ".example.com".
+
+              > **Caution.** Specify the list of subnets that are used on the nodes..
```
```suggestion
              > **Caution.** Specify the list of subnets that are used on the nodes.
```

z9r5 commented:
@Suselz The [PR](https://github.com/deckhouse/deckhouse/issues/8528) fixed the behavior you describe. Are really sure it is needed to make the notice in the documentation? Maybe DKP code has to be fixed?

Suselz commented:
> The https://github.com/deckhouse/deckhouse/issues/8528 fixed the behavior you describe. Are really sure it is needed to make the notice in the documentation? Maybe DKP code has to be fixed?

Yes, indeed. I got confirmation that it is a bug. I opened an issue #14433.

I corrected the documentation, corrected the wording when it is specifically required to specify the node networks in this parameter

deckhouse-BOaTswain commented:
Cherry pick PR [14439](https://github.com/deckhouse/deckhouse/pull/14439) to the branch [release-1.71](https://github.com/deckhouse/deckhouse/tree/release-1.71) successful!
